examples/platform/silabs/wifi/icd/ApplicationSleepManager.cpp

```diff
@@ -43,13 +43,17 @@ CHIP_ERROR ApplicationSleepManager::Init()
 void ApplicationSleepManager::OnCommissioningWindowOpened()
 {
     mIsCommissionningWindowOpen = true;
+#if ( ( defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1 ) )
     mWifiSleepManager->VerifyAndTransitionToLowPowerMode();
+#endif
 }
 
 void ApplicationSleepManager::OnCommissioningWindowClosed()
 {
     mIsCommissionningWindowOpen = false;
+#if ( ( defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1 ) )
```

Comment:
Shouldn't we still allow to go to low power on commissioning window closure? The description talks about the commissioning window open

Reply:
While PASE session establishment, commissioning window is getting closed and if sleep is enabled at this time wifi-connect is failing all the  times on NCP. So enabling sleep only after wifi connect.

Reply:
yes @mkardous-silabs this behavior has been verified. @chirag-silabs had also seen this and @bhmanda-silabs also saw that.

Reply:
This is going to prevent the NCP from going to sleep in a multi admin scenario in that case.

-> Commission the device and goes to sleep
-> Open commission window
-> let the window timeout

does the device go to sleep?

Reply:
Follow up task created: MATTER-4442. Merging for this release.